compiler/rustc_borrowck/src/diagnostics/region_name.rs

```diff
@@ -619,7 +619,7 @@ impl<'tcx> MirBorrowckCtxt<'_, 'tcx> {
                     // programs, so we need to use delay_span_bug here. See #82126.
                     self.infcx.tcx.sess.delay_span_bug(
                         hir_arg.span(),
-                        format!("unmatched subst and hir arg: found {kind:?} vs {hir_arg:?}"),
+                        format!("unmatched arg and hir arg: found {kind:?} vs {hir_arg:?}"),
```

Comment:
"arg and hir arg" -- maybe "generic param and hir arg"?

Reply:
well, it is a `ty::GenericArg` and a hir generic arg, the ty generic arg could be something like 3 which is not a param :thinking: the comment is somewhat :/ rn though